### PR TITLE
Add MARS

### PR DESCRIPTION
**Pull request by JeremyParish69**
-to frontier
-as registered in chain registry
-transfers in both directions validated (and explorer link is good)
-uses spot price with pool id 907 MARS/OSMO

### CODE REVIEW

**Comment by jonator** [comment]
@JeremyParish69 What else is needed?

**Comment by JeremyParish69** [comment]
> @JeremyParish69 What else is needed?

nothing else, just want to confirm that transferring works before merging

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -2545,10 +2545,18 @@ const chainInfos = (
         "https://explorer.dys.dysonprotocol.com/dyson/tx/{txHash}",
     },
     {
-      rpc: "https://testnet-rpc.marsprotocol.io/",
-      rest: "https://testnet-rest.marsprotocol.io/",
-      chainId: "ares-1",
-      chainName: "Mars Hub Testnet",
+      rpc: IS_TESTNET
+        ? "https://testnet-rpc.marsprotocol.io/"
+        : "https://rpc.marsprotocol.io:443",
```
Comment (sunnya97) [inline packages/web/config/chain-infos.ts:2550]:
```suggestion
        : "https://rpc.marsprotocol.io/",
```

File: packages/web/config/chain-infos.ts [hunk 2]
```diff
@@ -2545,10 +2545,18 @@ const chainInfos = (
...
+      rest: IS_TESTNET
+        ? "https://testnet-rest.marsprotocol.io/"
+        : "https://rest.marsprotocol.io:443",
```
Comment (sunnya97) [inline packages/web/config/chain-infos.ts:2553]:
```suggestion
        : "https://rest.marsprotocol.io/",
```

**Review by sunnya97 — APPROVED** [review]

**Pull request merged**